pyi.py

```diff
@@ -190,7 +192,7 @@ def CLASSDEF(self, node):
         self.popScope()
         self.addBinding(node, ClassDefinition(node.name, node))
 
-    def handleNodeDelete(self, node):
+    def handleNodeDelete(self, node) -> None:
         """Null implementation.
 
         Lets users use `del` in stubs to denote private names.
```

Comment:
I don't think we use this feature anymore in typeshed. I'm not even sure if type checkers support `del` statements in stubs.

Reply:
Yeah, I have no idea. It might be used by third-party stubs, though, you never know.